dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractJdbcTask.java

```diff
@@ -82,7 +93,28 @@ protected CSVFormat newCsvFormat(@Nonnull ResultSet rs) throws SQLException {
     Class<? extends Enum<?>> headerClass = getHeaderClass();
     if (headerClass != null) {
       format = format.withHeader(headerClass);
-      if (headerClass.getEnumConstants().length != rs.getMetaData().getColumnCount())
+      ResultSetMetaData metaData = rs.getMetaData();
+      if (isAssessment) {
+
+        HashSet<String> columnNames = new HashSet<>();
+
+        for (int i = 0; i < metaData.getColumnCount(); i++) {
+          columnNames.add(metaData.getColumnName(i));
+        }
+
+        String missingColumns =
+            Arrays.stream(headerClass.getEnumConstants())
+                .map(Enum::name)
+                .map(name -> CaseFormat.UPPER_CAMEL.to(CaseFormat.UPPER_UNDERSCORE, name))
```

Comment:
Can be done in the same .map()

Reply:
Seeing the method reference felt better. Are you strongly against this?

Reply:
It could simply have been:
.map(enumVal -> CaseFormat.UPPER_CAMEL.to(CaseFormat.UPPER_UNDERSCORE, enumVal.name()))

This way feels much better for me...  but it is optional.

Reply:
Done